scripts/superdesk-authoring/authoring.js

```diff
@@ -102,7 +102,7 @@
     }
 
     function stripHtml(item) {
-        var fields = ['headline', 'abstract'];
```

Comment:
so seems like this change is fine, but requires a backend fix toom, right? any specific format where it should be stripped @akintolga ?